nixos/tests/phpfpm.nix

```diff
@@ -0,0 +1,71 @@
+import ./make-test-python.nix ({ pkgs, lib, ... }: {
+  name = "phpfpm";
+  meta.maintainers = [ lib.maintainers.aanderse ];
+
+  machine =
+    { config, ... }:
+    { services.httpd.enable = true;
+      services.httpd.adminAddr = "admin@example.com";
+      services.httpd.extraModules = [ "proxy_fcgi" ];
+      services.httpd.extraConfig = ''
+        Alias /site1 /var/www/site1
+        <Directory /var/www/site1>
+          <FilesMatch "\.php$">
+            <If "-f %{REQUEST_FILENAME}">
+              SetHandler "proxy:unix:${config.services.phpfpm.pools.site1.socket}|fcgi://localhost/"
```

Comment:
what does that do? fallback to fcgi localhost?

Reply:
Simply following apache documentation: https://cwiki.apache.org/confluence/display/httpd/PHP-FPM

Reply:
This still doesn't explain what's up with the second line here. I think we should only configure it to (only) use the socket, don't we?

Reply:
Sorry for the lackluster answer, this is just the way I've always seen it done and never really thought too much about it. Here is the actual explanation of **why** it needs to be done that way: https://www.linuxquestions.org/questions/linux-server-73/understanding-apache-proxy-unix-sockets-with-pipe-localhost-4175627589/#post5843017